README.md

```diff
@@ -88,13 +114,15 @@ OpenTelemetry Collector
 
 ## Getting Started
 
-### With Tilt and Local Kubernetes
+Use Tilt to spin up all the required backends:
 
 ```shell
-Start Tilt in the project root to set up the local Kubernetes environment:
+# Start Tilt in the project root to set up the local Kubernetes environment:
```

Comment:
Setting the api key in .env is required before tilt up and should be documented somewhere here above.